app/lib/bill_parser.rb

```diff
@@ -8,10 +8,19 @@ class BillParser
   }
 
   class << self
-    def bills(page)
+    def latest_bills(page)
       PROPOSERS.flat_map do |key, proposer|
         rows = table_rows_without_header(page, proposer[:type])
-        build_bills(rows, proposer[:name], proposer[:id], latest_session_number(page))
+        build_bills(rows, proposer[:name], proposer[:id], current_session_number(page))
+      end
+    end
+
+    def old_bills(pages)
+      pages.flat_map do |page|
+        PROPOSERS.flat_map do |key, proposer|
+          rows = table_rows_without_header(page, proposer[:type])
+          build_bills(rows, proposer[:name], proposer[:id], current_session_number(page))
+        end
```

Comment:
この部分は`latest_bills`と同じに見えます。

```suggestion
        latest_bills page
```
もしかして、上のように書けたりしますか？

Reply:
@ledsun ご指摘ありがとうございます✨
ご提示の方法で書けます・・・！
そしてそれならbill_parserの中でlatest_bills, old_billsなどと分けなくとも、単にbillsメソッドだけ用意すれば良さそうです。修正します！

Reply:
@ledsun 修正いたしました！
https://github.com/chihaso/bill_watcher/pull/37/commits/65422cc76a5906c6f2f580c682b324f7a2f93cfd